data/dbconn.py

```diff
@@ -1133,11 +1186,17 @@ async def update_rounds(self, client, guild=None):
         curr = self.conn.cursor()
         curr.execute(query)
         data = curr.fetchall()
+        curr.close()
         for x in data:
             try:
                 guild = client.get_guild(x[0])
                 users = [guild.get_member(int(x1)) for x1 in x[1].split()]
                 handles = [self.get_handle(guild.id, user.id) for user in users]
+                temp = self.fetch_handles(guild.id,users)
+                if temp is not None:
+                    for handle in temp:
+                        if handle not in handles:
+                            handles.append(handle)
```

Comment:
after few lines the bot is fetching subs of all the handles in handles[] so it will also fetch the submissions of alts which are not needed so use this code only if we want to give a new problem and that too after that step

Reply:
ok, I will reposition this part